README.md

```diff
@@ -1,38 +1,139 @@
+
 # NiftyOne
 
 ![NiftyOne Mosaic](.github/static/niftyone_mosaic_view.png)
 
 NiftyOne is a platform for bulk visualization of large-scale neuroimaging datasets. It is built with [FiftyOne](https://docs.voxel51.com/).
 
-## Installation
+- [NiftyOne](#niftyone)
+  * [Installation](#installation)
+  * [Testing](#testing)
+  * [Usage](#usage)
+  * [Running NiftyOne on a remote cluster](#running-niftyone-on-a-remote-cluster)
+  * [FAQ](#faq)
+
 
+## Installation
+**1. Install [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html)**
 For stability, NiftyOne should be installed in it's own environment. For example,
 
 ```bash
 conda create -y -n niftyone python=3.10
 conda activate niftyone
 
 pip install -U pip
-pip install -r https://raw.githubusercontent.com/cmi-dair/niftyone/main/requirements.txt
 pip install git+https://github.com/cmi-dair/niftyone.git
 ```
-
+## Testing
```

Comment:
To me this feels a bit too in the weeds for the README. I suggest we keep the usage instructions minimal in the README and then move the more detailed tutorial content to a `TUTORIAL.md` file where we can freely go into details.